thirdPage.css

```diff
@@ -0,0 +1,107 @@
+body{
+    background-color: white;
+    padding:0px 40px 0px 40px;
+}
+.navbar{
+    margin-bottom:-20px;
+}
+#home-section{
+    background-image: url("http://eskipaper.com/images/snow-mountain-8.jpg");
+    color:white;
+    padding: 90px;
+}
+.btn-home{
+    margin-top:30px;
+    width: 150px;
+    height:auto;
+}
+.margin-bottom-50px{
+    margin-bottom:50px;
+}
+.text-about{
+    color:#96C44F;
+
+}
+.meet-the-team-margin{
```

Comment:
napravi posebne klase za margine, lakse ti je za sve. Napravis dje ti god treba margina klasu i upotrijebis je, makar imao 40 tih klasa. Dacu ti sliku kako to kod mene izgleda u jednom kodu. 
ako imas tipa 
margin-top-10{
margin-top: 10px;
}
Mozes da je upotrijebis dje ti god treba gornja margina od 10.